examples/github-notification-filters/global.d.ts

```diff
@@ -0,0 +1,9 @@
+import { TagsApi } from '@algolia/autocomplete-plugin-tags';
+
+import { NotificationFilter } from './types';
+
+declare module '@algolia/autocomplete-core' {
+  interface AutocompleteContext {
+    tagsPlugin?: TagsApi<NotificationFilter>;
+  }
+}
```

Comment:
This should allow us to type the context, and it does in my editor but only when the declaration is either in the `app.tsx` file, or when this file is open 🤷 Normally there's no need to add it to `tsconfig.json`, and when I do it break other things. Help appreciated!

Reply:
We've got the same issue with our other plugin contexts.

Reply:
Normally this should fix it per codebase, and it seems to, but there's a tooling thing that I can't put my finger on yet.

I will try to invetsigate this later on.